webapp/lib/app/configurator/tags/view.dart

```diff
@@ -257,9 +260,16 @@ class SampleMessagesTooltip {
     }
   }
 
-  String _conversationLinkFromMessageID(String messageID) {
-    var conversationID = messageID.replaceAll('nook-message-', '').substring(0, 52);
-    return "/converse/index.html?conversation-id=${conversationID}";
+  String _conversationLinkFromMessageID({String messageId, String tagId}) {
```

Comment:
I think we need to rename this now, maybe something like: `_linkToFilteredConversationView`?